zetkin/app.zetkin.org
# Spring Cleaning 2

This PR contains the following overall adjustments:
- Enable `no-trailing-spaces` linting rule
- Move Zetkin API object interfaces to `types/zetkin.ts ` module
- Rename components to remove unnecessary `Org*` prefixes
- Rename `data-test` attributes to `data-testid`
- Enforce alphabetical order of TS interface members

No adjustments for the following:
- Enable prefer-const linting rule _(Was already implemented)_
- Look over occurrences of `eslint-disable @typescript-eslint/no-non-null-assertion` and clean up usage of `!` _(Found no unnecessary assertions)_
- Verify typing for `props.children` everywhere (especially along with `<Flex/>`) _(More research needed)_
